.github/CODEOWNERS

```diff
@@ -551,3 +553,5 @@
 /yarn.lock @vojtechjelinek @kevinlee12
 /scripts/install_third_party_libs.py @vojtechjelinek @kevinlee12
 /.github/ @kevinlee12
+/.github/CODEOWNERS @DubeySandeep
```

Comment:
[I had left this comment earlier as well but I don't see this in the thread!]

You might have to do some changes in the lint test for CODEOWNER_IMPORTANT_PATHS. [else the test will fail]

Reply:
Ah, good catch. I've added that to the path list.